time    day             # The day this measurement is about.
float64 avg_temp        # Average temperature in F.
float64 max_temp        # Maximum reported temperature in F.
float64 min_temp        # Minimum reported temperature in F.